src/ontology/mondo-edit.obo

```diff
@@ -83900,9 +83900,13 @@ name: diabetic kidney disease
 def: "Progressive kidney disorder caused by vascular damage to the glomerular capillaries, in patients with diabetes mellitus. It is usually manifested with nephritic syndrome and glomerulosclerosis." [NCIT:C84417]
 synonym: "diabetic nephropathy" EXACT [NCIT:C84417]
 synonym: "DKD" EXACT ABBREVIATION [NCIT:C84417]
+synonym: "type 1 diabetes nephropathy" NARROW [MONDO:0005442]
```

Comment:
FYI - I changed the synonyms from exact to narrow, since they are more specific.
Everything else looks great.